### PR TITLE
Fixed format strings to match arguments

### PR DESCRIPTION
**Pull request by akallabeth**
Reviewed and replaced all %d specifiers to match proper type

### CODE REVIEW

File: libfreerdp/core/nla.c [hunk 1]
```diff
@@ -1818,9 +1818,9 @@ SECURITY_STATUS nla_decrypt_public_key_echo(rdpNla* nla)
 	{
 		WLog_ERR(TAG, "Could not verify server's public key echo");
 #if defined(WITH_DEBUG_NLA)
-		WLog_ERR(TAG, "Expected (length = %d):", public_key_length);
+		WLog_ERR(TAG, "Expected (length = %" PRIu32 "):", public_key_length);
```
[the remark this answers is not in the document]
Comment (akallabeth) [reply libfreerdp/core/nla.c:1821]: @mfleisz `%lu` is `64bit` on linux as `long` is `int64_t`

[the remark this answers is not in the document]
Comment (mfleisz) [reply libfreerdp/core/nla.c:1821]: @akallabeth yepp - that's why I don't think `PRIu32` is correct here - `%lu` is the right match for `public_key_length` (which is defined as `unsigned long`).

[the remark this answers is not in the document]
Comment (akallabeth) [reply libfreerdp/core/nla.c:1821]: @mfleisz no, it is defined in a windows compatible way: `typedef __uint32 ULONG, *PULONG;`

[the remark this answers is not in the document]
Comment (mfleisz) [reply libfreerdp/core/nla.c:1821]: @akallabeth then our `typedef` is wrong: https://learn.microsoft.com/en-us/windows/win32/winprog/windows-data-types

[the remark this answers is not in the document]
Comment (akallabeth) [reply libfreerdp/core/nla.c:1821]: @mfleisz no. different data models for `long` types.

[the remark this answers is not in the document]
Comment (mfleisz) [reply libfreerdp/core/nla.c:1821]: @akallabeth now I got you 👍

**Review by mfleisz — APPROVED** [review]

**Review by mfleisz — APPROVED** [review]

**Pull request merged**